src/main/java/DeadlineTest.java

```diff
@@ -0,0 +1,36 @@
+import duke.task.Deadline;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class DeadlineTest {
+
+    @Test
+    public void testStringConversion() {
+        Deadline deadline = new Deadline("project", LocalDate.of(2021, 9, 1));
+        assertEquals("[D][ ] project (by: Sep 1 2021)", deadline.toString());
+    }
+
+    @Test
+    public void stringConversion_doneTask_indicated() {
```

Comment:
Following the three part format "featureUnderTest_testScenario_expectedBehavior()" for writing names for test functions, you might want to rename `stringConversion_doneTask_indicated()`to make it clearer